### ISSUE TITLE
Add 'description' to packages

### ISSUE THREAD

**Issue by monsdar** [conan-io/conan#596]
Lastly the package index of conan is blowing up. Not all of those packages are self-explanatory, so having a bit more information directly available could help understand.

This can be as easy as adding a `description` member to the conanfile. Perhaps it would be good to also have a `short_description` with a fixed limit of characters.

**Comment by monsdar** [comment]
Example:
The package [7z_installer](https://www.conan.io/source/7z_installer/0.1/lasote/testing) contains the 7zip executable for a certain Windows version. What is its purpose, how is it meant to be used?

Of course I can visit the provided `url`, but wouldn't it be better to have some basic information directly available on the webpage?

**Comment by opilar** [comment]
Maybe it'd be more convinient to extract a short description from a description:
```python
class ZlibConan(ConanFile):
    name = "zlib"
    version = "1.2.8"
    description = """Data compression library.
Zlib uses deflate compression algorithm.
...
```

Where `Data compression library.` can be a short description.

**Comment by memsharded** [comment]
Agree with @opilar. Better just one field ``description``, than two, and using such typical approach of the first line. Also, better than using the class docstring, which could be used, but better let the docstring for developer purposes, about the internal of the recipe, and let the ``description`` be the public description for interested consumers.

**Comment by lasote** [comment]
Will be released in 0.17, today. Also in website will be shown in each package homepage (if present).
